### ISSUE TITLE
Issue with Get-MgUserAnalytic

### ISSUE THREAD

**Issue by Semptra** [microsoftgraph/msgraph-sdk-powershell#318]
Hello.
I've tried to use `Microsoft.Graph.Analytics` PowerShell module and `Get-MgUserAnalytic` cmdlet, but it constantly returns the same error:
```powershell
PS > Get-MgUserAnalytic -UserId "********-****-****-****-************"
Get-MgUserAnalytic : {"Message":"No HTTP resource was found that matches the request URI 'https://substrate.office.com:444/WeveB2/api/v2.0/users('********-****-****-****-************@********-****-****-****-************')'.","MessageDetail":"No type was found that matches the controller named 'users'."}
At line:1 char:1
+ Get-MgUserAnalytic -UserId "********-****-****-****-************" -De ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: ({ UserId = ********\u2026ExpandProperty =  }:<>f__AnonymousType5`3) [Get-MgUserAnalytic_Get], Exception
+ FullyQualifiedErrorId : UnknownError,Microsoft.Graph.PowerShell.Cmdlets.GetMgUserAnalytic_Get
```

The ID that I'm providing is valid and works with other cmdlets (for example, `Get-MgUserInsight` works fine).

`Micrsoft.Graph` version: 0.7.1

`PowerShell Core` version:

```powershell
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19041
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by peombwa** [comment]
Get-MgUserAnalytic is not a valid cmdlet. It calls the root `/analytics` path which is not valid.

It will be removed by #320 .

**Comment by peombwa** [comment]
This has been fixed in 0.9.1. `Update-Module Microsoft.Graph.PowerShell`.
